src/style.css

```
@@ -1 +1,68 @@
-/* Styles go here */
+* {
+  margin: 0;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  text-transform: uppercase;
+  margin-left: 40px;
+  margin-right: 40px;
+}
+
+.logo {
+  margin-top: 10px;
+}
+
+.nav {
+  display: flex;
+  flex-shrink: 0;
+}
+
+.nav_list {
+  list-style: none;
+  display: flex;
+  align-self: stretch;
+  flex-shrink: 0;
+}
+
+a {
+  position: relative;
+  display: flex;
+  text-decoration: none;
+  flex-shrink: 0;
+  flex-grow: 1;
+  margin: 10px;
+  color: black;
+  line-height: 60px;
+  margin-top: 0;
+}
+
+a:after {
+  position: absolute;
+  content: "";
+  bottom: 0;
+  left: 0;
+  height: 4px;
+  border-radius: 4px;
+  background-color: #00acdc;
+}
+
+a:active:after {
+  width: 100%;
+}
+
+a:hover {
+  color: #00acdc;
+}
+
+.is-active {
+  color: #00acdc;
+}
```

```suggestion
a:hover,
.is-active {
  color: #00acdc;
}
```
Стилі дублюються. Можна об'єднати